### ISSUE TITLE
Example crashes

### ISSUE THREAD

**Issue by jedi1150** [DeepARSDK/quickstart-android-java#31]
Hi!
I have an issue when trying to run example from https://developer.deepar.ai/downloads (3.1.0).
```log
2021-08-23 18:43:33.267 18204-18204/ai.deepar.deepar_example E/linker: library "/system/lib64/libOpenCL.so" ("/system/lib64/libOpenCL.so") needed or dlopened by "/data/app/ai.deepar.deepar_example-Ogc6SefRwrp8FzgKw_MIJg==/lib/arm64/libnative-lib.so" is not accessible for the namespace: [name="classloader-namespace", ld_library_paths="", default_library_paths="/data/app/ai.deepar.deepar_example-Ogc6SefRwrp8FzgKw_MIJg==/lib/arm64:/data/app/ai.deepar.deepar_example-Ogc6SefRwrp8FzgKw_MIJg==/base.apk!/lib/arm64-v8a", permitted_paths="/data:/mnt/expand:/data/data/ai.deepar.deepar_example"]
2021-08-23 18:43:33.996 18204-18204/ai.deepar.deepar_example A/libc: Fatal signal 11 (SIGSEGV), code 1, fault addr 0x0 in tid 18204 (.deepar_example)
```
But when i comment `deepAR.setRenderSurface` it no longer has an issue.
On a project with previous versions of deepar.aar, It doesn't happen.

It works without problems on Google Pixel, but on Google Pixel 3 crashes.

**Comment by jedi1150** [comment]
```log
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: Build fingerprint: 'google/blueline/blueline:12/SPB4.210715.011/7608474:user/release-keys'
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: Revision: 'MP1.0'
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: ABI: 'arm64'
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: Timestamp: 2021-08-24 12:27:17.943715084+0300
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: Process uptime: 0s
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: Cmdline: ai.deepar.deepar_example
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: pid: 12090, tid: 12090, name: .deepar_example  >>> ai.deepar.deepar_example <<<
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: uid: 10449
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: Cause: null pointer dereference
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x0  00000075413e6598  x1  000000773a9b35b1  x2  0000000000000200  x3  0000000000000000
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x4  0000007fec63dda9  x5  000000773a9b35d3  x6  616c507465476c63  x7  7344496d726f6674
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x8  0000000000000000  x9  a57a4620cf70128c  x10 0000000000004001  x11 0000000000000000
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x12 000000773a89ab94  x13 0000000100000000  x14 0000007fec63dcf8  x15 ffffffffffffffff
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x16 0000000000000030  x17 000000773a9852c0  x18 0000007739b7c000  x19 00000075413e6590
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x20 00000073f3175000  x21 00000073f31750b0  x22 00000073f31750a8  x23 00000073f3173250
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x24 00000075c13a5c58  x25 0000000000001ac6  x26 00000077395af000  x27 00000077395af000
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     x28 00000073f3173380  x29 0000007fec63e030
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:     lr  00000073f2d9acbc  sp  0000007fec63df00  pc  00000073f2d9accc  pst 0000000020000000
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG: backtrace:
2021-08-24 12:27:18.393 12211-12211/? A/DEBUG:       #00 pc 0000000000062ccc  /data/app/~~RRBuV8DdQtlbr8h_s105Jg==/ai.deepar.deepar_example-ZoEc0OiZV-GczojxP01CVw==/lib/arm64/libnative-lib.so (BuildId: cf30f7a71a79b125410af227247160f4309da832)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #01 pc 000000000005dd38  /data/app/~~RRBuV8DdQtlbr8h_s105Jg==/ai.deepar.deepar_example-ZoEc0OiZV-GczojxP01CVw==/lib/arm64/libnative-lib.so (BuildId: cf30f7a71a79b125410af227247160f4309da832)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #02 pc 000000000004a1a8  /data/app/~~RRBuV8DdQtlbr8h_s105Jg==/ai.deepar.deepar_example-ZoEc0OiZV-GczojxP01CVw==/lib/arm64/libnative-lib.so (Java_ai_deepar_ar_DeepAR_initializeAR+380) (BuildId: cf30f7a71a79b125410af227247160f4309da832)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #03 pc 0000000000330044  /apex/com.android.art/lib64/libart.so (art_quick_generic_jni_trampoline+148) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #04 pc 0000000000326764  /apex/com.android.art/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #05 pc 0000000000323b9c  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+312) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #06 pc 00000000003cf3b4  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<true, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+1328) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #07 pc 00000000003cdff4  /apex/com.android.art/lib64/libart.so (MterpInvokeDirectRange+220) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #08 pc 000000000033d214  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_direct_range+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #09 pc 00000000000d17ca  [anon:dalvik-classes.dex extracted in memory from /data/app/~~RRBuV8DdQtlbr8h_s105Jg==/ai.deepar.deepar_example-ZoEc0OiZV-GczojxP01CVw==/base.apk]
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #10 pc 0000000000349634  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.15880507524294088337)+628) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #11 pc 0000000000352ea4  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+124) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #12 pc 00000000003234bc  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+2664) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #13 pc 0000000000321518  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+436) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #14 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #15 pc 0000000000039f1c  /data/data/ai.deepar.deepar_example/code_cache/.overlay/base.apk/classes2.dex
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #16 pc 0000000000349634  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.15880507524294088337)+628) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #17 pc 0000000000352ea4  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+124) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #18 pc 00000000003234bc  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+2664) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #19 pc 0000000000464674  /apex/com.android.art/lib64/libart.so (MterpInvokeInterface+464) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #20 pc 000000000033d014  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_interface+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #21 pc 00000000003093f2  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #22 pc 0000000000321ac4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+1888) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #23 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #24 pc 0000000000307794  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #25 pc 0000000000321ac4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+1888) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #26 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #27 pc 0000000000306ca4  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #28 pc 0000000000349634  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.15880507524294088337)+628) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #29 pc 0000000000352ea4  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+124) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #30 pc 00000000003234bc  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+2664) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #31 pc 0000000000464674  /apex/com.android.art/lib64/libart.so (MterpInvokeInterface+464) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #32 pc 000000000033d014  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_interface+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #33 pc 0000000000330988  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #34 pc 0000000000321ac4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+1888) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #35 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #36 pc 000000000032d7a6  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #37 pc 00000000007895c4  /apex/com.android.art/lib64/libart.so (MterpInvokeDirect+1440) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #38 pc 000000000033cf14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_direct+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #39 pc 0000000000329c5a  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #40 pc 0000000000321ac4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+1888) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #41 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #42 pc 0000000000323a20  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #43 pc 0000000000464bb8  /apex/com.android.art/lib64/libart.so (MterpInvokeInterface+1812) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #44 pc 000000000033d014  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_interface+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #45 pc 00000000002c7d00  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #46 pc 0000000000321ac4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+1888) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #47 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #48 pc 00000000002c843e  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #49 pc 00000000003cade4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtualRange+2044) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #50 pc 000000000033d114  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual_range+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #51 pc 00000000002c872c  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #52 pc 0000000000321ac4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+1888) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #53 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #54 pc 00000000002c7e82  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #55 pc 0000000000464bb8  /apex/com.android.art/lib64/libart.so (MterpInvokeInterface+1812) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #56 pc 000000000033d014  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_interface+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #57 pc 000000000042326c  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #58 pc 000000000078a828  /apex/com.android.art/lib64/libart.so (MterpInvokeStatic+1124) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #59 pc 000000000033cf94  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_static+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #60 pc 00000000004230e0  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #61 pc 0000000000321ac4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+1888) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #62 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #63 pc 000000000044c952  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #64 pc 000000000078a828  /apex/com.android.art/lib64/libart.so (MterpInvokeStatic+1124) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #65 pc 000000000033cf94  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_static+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #66 pc 000000000044cf98  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #67 pc 000000000078ab20  /apex/com.android.art/lib64/libart.so (MterpInvokeStatic+1884) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #68 pc 000000000033cf94  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_static+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #69 pc 00000000001c7d6a  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #70 pc 0000000000349634  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.15880507524294088337)+628) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #71 pc 0000000000348950  /apex/com.android.art/lib64/libart.so (artQuickToInterpreterBridge+1184) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #72 pc 0000000000330178  /apex/com.android.art/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #73 pc 00000000003269e8  /apex/com.android.art/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #74 pc 000000000037cb48  /apex/com.android.art/lib64/libart.so (_jobject* art::InvokeMethod<(art::PointerSize)8>(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jobject*, _jobject*, unsigned long)+636) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #75 pc 000000000037c8a4  /apex/com.android.art/lib64/libart.so (art::Method_invoke(_JNIEnv*, _jobject*, _jobject*, _jobjectArray*)+52) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #76 pc 00000000000a7f74  /apex/com.android.art/javalib/arm64/boot.oat (art_jni_trampoline+132) (BuildId: d624782d10c74d4ffdf1804a53af9d308747e43c)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #77 pc 0000000000326764  /apex/com.android.art/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #78 pc 0000000000323b9c  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+312) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #79 pc 000000000032335c  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+2312) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #80 pc 0000000000321518  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtual+436) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #81 pc 000000000033ce14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #82 pc 00000000002476b2  /system/framework/framework.jar
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #83 pc 0000000000349634  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.15880507524294088337)+628) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #84 pc 0000000000348950  /apex/com.android.art/lib64/libart.so (artQuickToInterpreterBridge+1184) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #85 pc 0000000000330178  /apex/com.android.art/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #86 pc 00000000007cfbc8  /system/framework/arm64/boot-framework.oat (com.android.internal.os.ZygoteInit.main+2232) (BuildId: c7a40d76edbf058f0d9755f6eee7f2d8808067ae)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #87 pc 00000000003269e8  /apex/com.android.art/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #88 pc 0000000000489450  /apex/com.android.art/lib64/libart.so (art::JValue art::InvokeWithVarArgs<_jmethodID*>(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jmethodID*, std::__va_list)+468) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #89 pc 000000000065bff0  /apex/com.android.art/lib64/libart.so (art::JNI<true>::CallStaticVoidMethodV(_JNIEnv*, _jclass*, _jmethodID*, std::__va_list)+268) (BuildId: 50c25b4aa1972771dac15f87d726a606)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #90 pc 00000000000aeac4  /system/lib64/libandroid_runtime.so (_JNIEnv::CallStaticVoidMethod(_jclass*, _jmethodID*, ...)+120) (BuildId: f076ab3f676ff38bc63fd12d52807ed0)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #91 pc 00000000000ba05c  /system/lib64/libandroid_runtime.so (android::AndroidRuntime::start(char const*, android::Vector<android::String8> const&, bool)+836) (BuildId: f076ab3f676ff38bc63fd12d52807ed0)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #92 pc 000000000000258c  /system/bin/app_process64 (main+1336) (BuildId: 4ecb2c57d6667e4a94a73568ce392d22)
2021-08-24 12:27:18.394 12211-12211/? A/DEBUG:       #93 pc 00000000000488c8  /apex/com.android.runtime/lib64/bionic/libc.so (__libc_init+96) (BuildId: 16a146efb5b048d744b674e300d5bcaf)
```

**Comment by JelenaKB** [comment]
Thank you for letting us know, we are working on a patch for this issue.

**Comment by JelenaKB** [comment]
This should be fixed in SDK v3.1.2
